src/components/MediaPlayerCarousel.tsx

```diff
@@ -109,28 +111,25 @@ export class MediaPlayerCarousel extends React.PureComponent<Props, State> {
 
   _attemptBoost = () => {
     ReactNativeHapticFeedback.trigger('impactHeavy', HapticOptions)
-    this.explosion && this.explosion.start()
-    this.setState({ boostPaymentLoading: true }, () => {
-      (async () => {
-        try {
-          const { nowPlayingItem } = this.global.player
-
-          const { errors, transactions, totalAmountPaid } = await sendBoost(nowPlayingItem)
-
-          this.setState({ boostPaymentLoading: false })
-          this.setGlobal({
-            bannerInfo: {
-              show: true,
-              description: translate('Boost Sent'),
-              errors,
-              transactions,
-              totalAmount: totalAmountPaid
-            }
+    this.setState({ boostIsSending: true }, () => {
+      this.explosion && this.explosion.start()
+      try {
+        const { nowPlayingItem } = this.global.player
+        sendBoost(nowPlayingItem)
+        setTimeout(() => {
+          this.setState({
+            boostIsSending: false,
+            boostWasSent: true
+          }, () => {
+            setTimeout(() => {
+              this.setState({ boostWasSent: false })
+            }, 4000)
           })
-        } catch (error) {
-          Alert.alert(translate('Boost Pay Error'), error.message)
-        }
-      })()
+        }, 1000)
+      } catch (error) {
```

Comment:
Without the await on sendBoost, are you sure the catch actually works?

Reply:
Oh wow, it doesn't throw the error without the await. I don't think we need the error handler now, so I'll remove the try/catch.